qtox.pro

```diff
@@ -312,6 +312,35 @@ contains(ENABLE_SYSTRAY_GTK_BACKEND, NO) {
 }
 }
 
+# The snorenotify backend implements a notification system compatible with many systems
+!android {
+contains(ENABLE_NOTIFICATION_SNORE_BACKEND, YES) {
+    DEFINES += ENABLE_NOTIFICATION_SNORE_BACKEND
+
+    unix {
+        INCLUDEPATH += "/usr/local/include/snore/core"
+        equals(QT_ARCH, x86_64) {
+            INCLUDEPATH += "/usr/local/lib64/libsnore-qt5/include"
+            INCLUDEPATH += "/usr/local/lib/x86_64-linux-gnu/libsnore-qt5/include"
+        }
+        else {
+            INCLUDEPATH += "/usr/local/lib/libsnore-qt5/include"
+            INCLUDEPATH += "/usr/local/lib/i386-linux-gnu/libsnore-qt5/include"
+        }
+
+        LIBS += -L/usr/local/lib64/ -lsnore-qt5 -lsnoresettings-qt5
```

Comment:
Well, I guess that's where they pointed on TheSpiritXIII's box. `/usr/local/` is where snorenotify's cmake puts libs by default too. Probably some distro differences are at play, but I can change this to `/usr/include/libsnore/` for includes and remove `/usr/local/lib*` references altogether.